Update readme, small corrections and emojis (#2)
ecosia/pycon22-prometheus-workshop: README.md

We will do this using the time-series database system [Prometheus](https://prometheus.io), which uses a "pull" method to extract data from running applications. This means that the applications need to "export" their data, so that Prometheus is able to "scrape" the metric data from them. This is typically done via an HTTP endpoint (`/metrics`, by convention).

We will use the [Prometheus Python client library](https://github.com/prometheus/client_python) to track metrics in our code.

### Agenda

* [Section 1: Exposing metrics](#section-1:-exposing-metrics)
* [Section 2: Creating custom metrics](#section-2:-creating-custom-metrics)
* [Section 3: Scraping Metrics with Prometheus and creating Dashboards with Grafana](#section-3:-scraping-metrics-with-prometheus-and-creating-dashboards-with-grafana)
* [Bonus Material: Histograms in Prometheus](#bonus-material:-histograms-in-prometheus)

### Prerequisites

> For this workshop you will need [Python 3](https://installpython3.com/), [Poetry](https://python-poetry.org/docs/#installation), [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/) running on your machine. *(on mac os docker-compose is by default installed with Docker)*

Reviewer: Ah okay, good that I see this! I was going to add it as I was surprised it wasn't there yet. Makes sense <3

---

### Section 2: Creating custom metrics 🔧

---

> Now that we are able to expose metrics we want to also be able to create them. Prometheus has a few different data types, but the most straight forward is a `Counter` - this is a counter which always goes up, and can be used to track, for example, the number of requests received (you can then divide this unit over time to calculate requests per second). To create a `Counter`, import it from the Prometheus Python client and instantiate it.

Reviewer: ```suggestion
Now that we can expose metrics, we need to create them. Prometheus has a few different data types, but the most straight forward is a `Counter` - this is a counter which always goes up, and can be used to track, for example, the number of requests received (you can then divide this unit over time to calculate requests per second). To create a `Counter`, import it from the Prometheus Python client and instantiate it.
```
